source/schema.txt

```diff
@@ -338,3 +338,424 @@ To delete a location filter from the map:
 .. |query-build-ref| replace:: :ref:`build-query`
 
 .. |query-build-type| replace:: :ref:`build-query`
+
+Export Your Schema
```

Comment:
I should've mentioned this earlier, but we probably want this on a sub-page of the main Schema Analysis page. This page was long already and adding this section certainly isn't helping the cause.